project-management/iteration-creation.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Iteration creation
+description: These are the steps to follow when creating a new iteration
+---
+
+## Before the iteration starts
+### Create a milestone
+
+Go to your squads repo and create a milestone titled: Iteration {number}-{week} {date from} - {date to} for example: Iteration 18-47 20th November - 3rd December.
+
+### Create a small task epic
+
+Create a new epic on the squad repo and attach it to the new milestone.
+
+### Set up the backlog
+
+Populate the "Proposed iteration" area in the backlog spreadsheet with high-level rows describing work for the upcoming iteration
+
+Move all incomplete epics down from the "Previous iteration" to the "Current iteration" section. Apart from the small tasks epic. Instead reassign all small tasks to the new small tasks in this iteration.
+
+### Create epics for every proposed backlog item
+
+Title the epic the same as the title in the backlog. Assign them to the appropriate milestone.
+
+Create and estimate broken down tasks for each epic in the next milestone. Assign them to the appropriate next milestone and set their pipeline to "Triaged".
+
+## At the start of the iteration
+
+Rename the sections of the backlog from "Current iteration:" to "Previous iteration". Then rename "Next iteration" to "Current iteration". Then create a new row titled "Next iteration" below "Current iteration".
+
+This should result in the "Previous iteration" being 100% complete. You may now hide the iteration rows.
+
+Pull up the epics that are proitised from this iteration.
+
+Be sure to check the "sum" cells are set currently.
+
+Hide the content of the completed iteration.
+
+Ensure all rows are correctly broken into issues with estimations.
+
+Check the number of points in the new iteration are achievable, if not, remove or add rows. This will inform the squad how much they take on as small tasks.
+
+### Assinging issues to small tasks
+
+Once you have completed you planning and know how many points you can still take on in the iteration. Scroll though the Triaged pipeline and choose a number of high, medium and low issues.
+
+I would suggest a ratio of 3 high, 2 medium and 1 small.
```

Comment:
```suggestion
A suggested ratio might be 3 high, 2 medium and 1 small.
```

Reply:
We don't typically attribute an author to docs so I've changed this to remove the first person tense.